lib/components/narrative/trip-tools.js

```diff
@@ -32,7 +33,14 @@ class TripTools extends Component {
           if (reactRouterConfig && reactRouterConfig.basename) {
             startOverUrl += reactRouterConfig.basename
           }
-          buttonComponents.push(<LinkButton icon='undo' text='Start Over' url={startOverUrl} />)
+          buttonComponents.push(
+            // FIXME: The Spanish string does not fit in button width.
+            <LinkButton
+              icon='undo'
+              text={intl.formatMessage({id: 'components.TripTools.startOver'})}
```

Comment:
Can `FormattedMessage` be used here instead?

Reply:
Is this tooltip still working regardless of formatting method used? I am not sure if something broke here.

Reply:
Yes, good catch. Fixed in https://github.com/opentripplanner/otp-react-redux/pull/430/commits/666ae9f14391c85fd7e2215b451840a93755aaec.